cluster-autoscaler/estimator/binpacking_estimator_test.go

```diff
@@ -79,10 +79,12 @@ func makePods(cpuPerPod int64, memoryPerPod int64, hostport int32, maxSkew int32
 	for i := 0; i < podCount; i++ {
 		pods = append(pods, pod)
 	}
-	return pods
+	return PodEquivalenceGroup{
+		Pods: pods,
+	}
 }
 
-func makeNode(cpu int64, mem int64, name string, zone string) *apiv1.Node {
+func makeNode(cpu, mem, podCount int64, name string, zone string) *apiv1.Node {
```

Comment:
Similar to the above. Could this use `BuildTestNode` instead? https://github.com/kubernetes/autoscaler/blob/7184d23e62367c57b2f0d66203a5d7c7a7eb661e/cluster-autoscaler/utils/test/test_utils.go#L215

Missing params can be added via option pattern. Right now it is hard to tell what `makeNode(1, 2, 3, "4", "5")` is doing without referring back here.

Reply:
Done

Reply:
Is it? I still see `makeNode` not `BuildTestNode` here.

Reply:
Updated to use `BuildTestNode` directly.